app/Controllers/ReservationsAdminController.php

```
@@ -296,10 +296,10 @@ public function store(Request $request, Response $response, mysqli $db): Respons
             $integrity = new \App\Support\DataIntegrity($db);
             $integrity->recalculateBookAvailability($libroId);
 
-            return $response->withHeader('Location', '/admin/prenotazioni?created=1')->withStatus(302);
+            return $response->withHeader('Location', '/admin/reservations?created=1')->withStatus(302);
         } else {
             $stmt->close();
-            return $response->withHeader('Location', '/admin/prenotazioni/crea?error=save_failed')->withStatus(302);
+            return $response->withHeader('Location', '/admin/reservations/create?error=save_failed')->withStatus(302);
         }
```

PHPMD (2.15.0)

[error] The method store uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)